src/main/java/com/github/fabienrenaud/jjb/stream/UsersStreamSerializer.java

```diff
@@ -1149,4 +1156,90 @@ public String underscore_java(final Users obj) throws IOException {
         }
         return jso;
     }
+
+    @Override
+    public Value purejson(Users obj) throws IOException {
+        final Map<Parser.Value, Parser.Value> map = new LinkedHashMap<>();
+        final List<Parser.Value> arr = new ArrayList<>();
+        if (obj.users != null) {
+            for (User u : obj.users) {
+                arr.add(purejson(u));
+            }
+            map.put(Value.string("users"), Value.arr(arr));
+        }
+        return Value.json(map);
+    }
+    
+    private Parser.Value purejson(final User u) throws IOException {
+        final Map<Parser.Value, Parser.Value> map = new LinkedHashMap<>();
+        
+        if (u._id != null) {
+            map.put(Value.string("_id"), Value.string(u._id));
+        }
+        map.put(Value.string("index"), Value.integer(u.index));
+        if (u.guid != null) {
+            map.put(Value.string("guid"), Value.string(u.guid));
+        }
+        map.put(Value.string("isActive"), Value.bool(u.isActive));
+        if (u.balance != null) {
+            map.put(Value.string("balance"), Value.string(u.balance));
+        }
+        if (u.picture != null) {
+            map.put(Value.string("picture"), Value.string(u.picture));
+        }
+        map.put(Value.string("age"), Value.integer(u.age));
+        if (u.eyeColor != null) {
+            map.put(Value.string("eyeColor"), Value.string(u.eyeColor));
+        }
+        if (u.name != null) {
+            map.put(Value.string("name"), Value.string(u.name));
+        }
+        if (u.gender != null) {
+            map.put(Value.string("gender"), Value.string(u.gender));
+        }
+        if (u.company != null) {
+            map.put(Value.string("company"), Value.string(u.company));
+        }
+        if (u.email != null) {
+            map.put(Value.string("email"), Value.string(u.email));
+        }
+        if (u.phone != null) {
+            map.put(Value.string("phone"), Value.string(u.phone));
+        }
+        if (u.address != null) {
+            map.put(Value.string("address"), Value.string(u.address));
+        }
+        if (u.about != null) {
+            map.put(Value.string("about"), Value.string(u.about));
+        }
+        if (u.registered != null) {
+            map.put(Value.string("registered"), Value.string(u.registered));
+        }
+        map.put(Value.string("latitude"), Value.number(u.latitude));
+        map.put(Value.string("longitude"), Value.number(u.longitude));
+        if (u.tags != null) {
+            final List<Parser.Value> jsarr = new ArrayList<>();
```

Comment:
init the capacity of the arraylist of u.tags.size()

same for other arraylists.